docs/updating-the-docs.md

```diff
@@ -48,3 +46,26 @@ Examples are included in the markdown files using the [pymdown snippet notation]
 Make sure you commit both the changes to the `examples` and `databuilder/snippets` directories.
 
 If you are updating multiple examples it might be easier to the extraction step happen each time you change the python modules, you can do this with `just databuilder/watch-examples`.
+
+
+## Updating Data Builder Backend and Contract documentation
```

Comment:
Nitpick: Titles in this section seem not to capitalize the first letter of each word. _Data Builder_ is an exception, because it's a proper noun (cf. common noun). By this reasoning, I think that _Backend_ and _Contract_ should be _backend_ and _contract_.